### PR TITLE
Update django support

### PR DESCRIPTION
**Pull request by hramezani**
## Problem

* Remove Django 3.1 from Python 3.10 env in tox.

 * Confirm support for Django 4.0

* Add USE_TZ setting.

## Solution

Explain the solution that has been implemented, and what has been changed.

## Commandments

- [ ] Write PEP8 compliant code.
- [ ] Cover it with tests.
- [ ] Update `CHANGES.rst` file to describe the changes, and quote according issue with `GH-<issue_number>`.
- [ ] Pay attention to backward compatibility, or if it breaks it, explain why.
- [ ] Update documentation (if relevant).

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -47,6 +47,7 @@ def long_desc(root_path):
         'Framework :: Django :: 2.2',
         'Framework :: Django :: 3.1',
         'Framework :: Django :: 3.2',
+        'Framework :: Django :: 4.0',
     ],
     zip_safe=False,
     tests_require=['Django>=2.2'],
```
Comment (adamchainz) [inline setup.py:53]: Shouldn't `install_requires` also say `>=2.2` ? I don't think there's even a need to put `django` in `tests_require` if it's in `install_requires`.

File: tox.ini [hunk 2]
```diff
@@ -1,8 +1,8 @@
 [tox]
 envlist =
-    py{36,37,38,39}-dj{22,31,32}
-    py310-dj{31,32}
-    py{38,39,310}-djmain
+    py{36,37,38,39}-dj{22,31}
+    py{36,37,38,39,310}-dj32
+    py{38,39,310}-dj{4.0,main}
```
Comment (adamchainz) [inline tox.ini:5]: I don't think this matches the below `dj40` dependency declaration with the `.`, it should be removed:

```suggestion
    py{38,39,310}-dj{40,main}
```

Comment (hramezani) [reply tox.ini]: Yes, you are right. it was a typo!

File: setup.py (hunk 1 above)
Comment (hramezani) [reply setup.py]: Good point. I update the `install_requires` and removed `tests_require`

**Comment by hramezani** [comment]
Thanks @adamchainz for your review!

**Review by MRigal — APPROVED** [review]
LGTM! Good that we had no test or setup depending on pytz. Thanks @hramezani !

**Pull request merged**